src/popup/components/TransactionInfoDetailsParty.vue

```diff
@@ -10,6 +10,7 @@
       class="name"
     >
       <Truncate
+        v-if="txParty.name || txParty.label"
```

Comment:
It can't be resolved like that - the design will break. Check out the solution from SW-493.
https://github.com/aeternity/superhero-wallet/pull/1763/files#diff-81e706951f062c4bd205ee4449b419270ae942e6161be2d6936446a537c0da64R14